processor/geoipprocessor/metadata.yaml

```diff
@@ -3,6 +3,6 @@ type: geoip
 status:
   class: processor
   stability:
-    development: [traces, metrics, logs]
+    alpha: [traces, metrics, logs]
```

Comment:
I believe we want to add `contrib` to the list of distributions now, and then open a PR against the releases repo as explained [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#releasing-new-components).

Reply:
Good catch! Added in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34788/commits/c9be10ffb294a5b4d8afccffb98337a0d214a697

Reply:
PR in the releases repository https://github.com/open-telemetry/opentelemetry-collector-releases/pull/646